chain-alerter/src/slot_time_monitor.rs

```diff
@@ -24,11 +23,9 @@ pub const DEFAULT_SLOW_SLOTS_THRESHOLD: f64 = 1.10;
 pub const DEFAULT_FAST_SLOTS_THRESHOLD: f64 = 0.94;
 
 /// The default maximum block buffer size.
+/// The check interval is approximately 6 times this number (the target block time is 6 seconds).
```

Comment:
There is no longer a check interval, or is it now implicit?

Reply:
It's now implicit, determined by `DEFAULT_MAX_BLOCK_BUFFER`.